modules/admin_manual/pages/enterprise/authentication/kerberos.adoc

```diff
@@ -0,0 +1,84 @@
+= Kerberos
+:toc: right
+:description: Kerberos is an authentication protocol and authenticates two hosts using the shared secret authentication technique. The main goal of Kerberos is to enable application authentication without the need to transmit user passwords.
+
+== Introduction
+
+{description} It will share a secret among the two hosts that only them and Key Distribution Center (KDC) knows. Kerberos builds a secure communication link between two trusted hosts across an untrusted network like the Internet using tickets.
```

Comment:
```suggestion
{description} It will share a secret among the two hosts that only they and the Key Distribution Center (KDC) know. Kerberos builds a secure communication link between two trusted hosts across an untrusted network like the Internet, using tickets.
```